CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 #### Dependency updates
 
+- Following packages updated
+  - `OpenTelemetry.ResourceDetectors.Azure` from `1.0.0-beta.4` to `1.0.0-beta.5`,
```

Comment:
@rajkumar-rangaraj, potential follow up issue - introduce here support for Azure Container Apps Resource Detector. What is you plan for it?

Ref: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/9832096f33783e2488176d804ae457d10318a6c5/src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md#100-beta5

Reply:
If we are interested, we could include that later.